README.md

```diff
@@ -53,12 +55,16 @@ may be out of date, the source of truth is whether the nearest anchor to a norma
 
 1. Find an untested (or undertested) normative statement in the [EPUB 3.3](https://www.w3.org/TR/epub-33/) or
    [EPUB Reading Systems 3.3](https://www.w3.org/TR/epub-rs-33/) specs to test — that is, a statement whose anchor element
-   does not have a `data-tests` attribute. Claim it in the
-   [spreadsheet of EPUB 3.3 Tests](https://docs.google.com/spreadsheets/d/13wgptApSVbQbYzi2L5VXe3D_1QLzJOTGySUAtOxn4sA/edit).
+   does not have a `data-tests` attribute. It may help to look at the (non-authoritative)
+   [spreadsheet of EPUB 3.3 Tests](https://docs.google.com/spreadsheets/d/13wgptApSVbQbYzi2L5VXe3D_1QLzJOTGySUAtOxn4sA/edit). 
+
+1. Claim the normative statement by [creating an issue](https://github.com/w3c/epub-tests/issues/new) in the
+   [w3c/epub-tests](https://github.com/w3c/epub-tests/) repo. For example, see the issue
+   [Test obfuscated resources (fonts)](https://github.com/w3c/epub-tests/issues/39).
 
-1. If you are an owner of [w3c/epub-tests](https://github.com/w3c/epub-tests/), create a branch on
-   that repo for your new test. Otherwise, fork the repo and create a branch on your fork. (It's
-   easier for reviewers to clone a PR to validate the test if it's in the original repo.)
+1. If you are an owner of [w3c/epub-tests](https://github.com/w3c/epub-tests/), create a branch on that repo for your new
+   test. Otherwise, fork the repo and create a branch on your fork. (It's easier for reviewers to clone a PR to validate the
+   test if it's in the original repo.)
 
 1. Within the branch, copy the
    [test template](https://github.com/w3c/epub-tests/tree/dir/tests/reading-systems/XX-Test-Template/epub-template)
```

Comment:
The URLs for the templates below are outdated; it should be `https://github.com/w3c/epub-tests/tree/main/tests/xx-epub-template` and `https://github.com/w3c/epub-tests/tree/main/tests/xx-fixed-layout-template`

Reply:
Oops, thanks! Addressed in https://github.com/w3c/epub-tests/pull/40/commits/2cf59e3cd8f47a43012e699df6c7d5771e3ab616.